plugins/@grouparoo/customerio/src/lib/export/exportProfile.ts

```diff
@@ -19,21 +20,27 @@ export const exportProfile: ExportProfilePluginMethod = async ({
 
   const customerId = newProfileProperties["customer_id"];
   const oldCustomerId = oldProfileProperties["customer_id"];
+  let newCustomer = null;
 
   if (!customerId) {
     throw new Error(`newProfileProperties[customer_id] is a required mapping`);
   }
-
-  if (oldCustomerId !== undefined && customerId !== oldCustomerId) {
-    // Must delete old customer if ID has changed
-    await client.destroy(oldCustomerId);
-  }
+  newCustomer = await client.getCustomer(customerId);
 
   if (toDelete) {
-    await client.destroy(customerId);
+    const oldCustomer = await client.getCustomer(oldCustomerId);
```

Comment:
Just to be sure, their API doesn't throw an error here if `oldCustomerId` is undefined, right?

Reply:
nope, It returns 404 which is treated by the client class.